db/schema.rb

```diff
@@ -11,27 +11,26 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160629045209) do
+ActiveRecord::Schema.define(version: 20160804083238) do
```

Comment:
I think our schemas between dev environments are actually different. This might be a difference in postgres version or the way our databases were created?